app/classifier/tasks/survey/editor.cjsx

```diff
@@ -50,6 +51,26 @@ module.exports = createReactClass
       <p><small>You can Apply changes after adding an individual csv file or many csv files.</small></p>
       <hr />
 
+      {if !@props.pfeLab  # show the instruction input if in FEM lab (don't show in PFE lab)
+        <div>
+          <AutoSave resource={@props.workflow}>
+            <label htmlFor="#{@props.taskPrefix}.instruction">
+              <span className="form-label">Main text</span>
+            </label>
+            <br />
+            <textarea
+              id="#{@props.taskPrefix}.instruction"
+              name="#{@props.taskPrefix}.instruction"
+              value={@props.task.instruction}
+              className="standard-input full"
+              onChange={handleInputChange.bind @props.workflow}
+            />
+          </AutoSave>
+          <small className="form-help">Describe the task, or ask the question, in a way that is clear to a non-expert. You can use markdown to format this text.</small>
```

Comment:
HTML `legend` is a single text string so any formatting here will be totally ignored by assistive technology, and would output invalid HTML in the browser. See https://github.com/zooniverse/front-end-monorepo/issues/2578

Reply:
The instructions for transcription tasks are probably the worst example of this on our site.

Reply:
`<input type=“text”>` would be more appropriate than a text area, to prevent people from creating instructions with line breaks.

Reply:
I agree - a task `instruction` or `question` should be limited. Changing the input from a textarea to a text input is a great idea and setting a length limit certainly would help! I'll open a related PFE issue, because we'll be using the PFE Project Builder for the foreseeable future. The issue can serve as a compliment to the FEM Issue 2578 linked. 

But I think I'll hold off on changing it in this PR for consistency with other task editors if that's ok? Not ideal, but then when we address the larger issue (FEM Issue 2578 and the PFE Issue I'll open) we can change everything together and consistently.

Reply:
That’s fine. The example project text looks good to me. 

We have got a blind volunteer who uses survey tasks in NVDA, and browser stack supports NVDA now so we can do some testing with that, to see how the instruction is announced and whether it is correctly linked to the survey task controls.